WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1601,6 +1601,16 @@ CreateContactGroupDialogTable   = @FullWidth@
 CreateContactGroupDialogCell    = padding:0 0 8px 0;
 CreateContactGroupDialogInput   = @FullWidth@
 
+#######################################################################
+#   Grant Rights Dialog
+#######################################################################
+
+GrantRightsDialogOuterContainer     = background-color:@lighten(SecondaryBrandingColor,94)@;
```

Comment:
Yes, discussion with UX will be good here.
But, what I feel is that  different modals differ in their taste.
Some are functional (ex. attach dialog), others are informative (Key board shortcut ex), others are for grating rights etc.
May be for UX guys, idea is to have some differentiation in the modal  for the same.
Just a thought.
Worth to bring this up with UX.